src/Mono.Android/Mono.Android.targets

```diff
@@ -365,14 +365,20 @@
         Command="$(ManagedRuntime) $(ManagedRuntimeArgs) $(_Mdoc) --debug update --use-docid --delete $(_Libdir) $(_ImportXml) $(_Output) $(_DocTypeArgs) $(_FxConfig) $(_Lang)"
         WorkingDirectory="$(MSBuildThisFileDirectory)"
     />
+    <!-- Ensure updated docs can also be exported to msxdoc format -->
+    <Exec
+        Command="$(ManagedRuntime) $(ManagedRuntimeArgs) $(_Mdoc) --debug export-msxdoc -o &quot;$(_LogPrefix)-Export.xml&quot; &quot;$(XamarinAndroidSourcePath)external/android-api-docs/docs/Mono.Android/en/&quot;"
+        WorkingDirectory="$(MSBuildThisFileDirectory)"
+    />
   </Target>
 
   <Target Name="_GenerateApiDocsDiff">
-    <PropertyGroup>
-      <_DiffFile>$(XamarinAndroidSourcePath)bin/Build$(Configuration)/UpdateApiDocs$([System.DateTime]::Now.ToString ("yyyyMMddTHHmmss")).diff</_DiffFile>
-    </PropertyGroup>
     <Exec
-        Command="git diff --output=&quot;$(_DiffFile)&quot;"
+        Command="git add ."
+        WorkingDirectory="$(XamarinAndroidSourcePath)external/android-api-docs"
+    />
+    <Exec
+        Command="git diff --cached --output=&quot;$(_LogPrefix).diff&quot;"
         WorkingDirectory="$(XamarinAndroidSourcePath)external/android-api-docs"
     />
```

Comment:
In other places we use:
https://github.com/xamarin/xamarin-android/blob/d8f8486718072b040d27a0955a5778e2618116f3/build-tools/scripts/XAVersionInfo.targets#L43-L44

It's probably worth using these variables in case we ever change them repo-wide. Maybe in some cases, someone would override to `/opt/homebrew/bin/git`?

Reply:
You might also be able to use our `<Git/>` task instead of `<Exec/>`, but only do that if it is easier:
https://github.com/xamarin/xamarin-android/blob/d8f8486718072b040d27a0955a5778e2618116f3/build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/Git.cs#L15

Reply:
Thanks, should be easy enough to use `<Git/>` instead as Mono.Android.targets already references a task from `$(PrepTasksAssembly)`.